pkgs/top-level/all-packages.nix

```
@@ -411,6 +411,8 @@ with pkgs;
 
   catatonit = callPackage ../applications/virtualization/catatonit { };
 
+  catppuccin-catwalk = with python3Packages; toPythonApplication catppuccin-catwalk;
```

```suggestion
  catppuccin-catwalk = callPackage ../development/tools/misc/catppuccin-catwalk { };
```
I'm not sure if this is the right place, but please move it from python-module.